### PR TITLE
chore: renaming to seedPhrase when phrase is used

### PR DESCRIPTION
**Pull request by hanwencheng**
Naming is the most difficult problem.

Renaming `seed` to `seedPhrase` if phrase is used.

In schema we use `encryptedSeed` since password will also used to secure that in the future version.

### CODE REVIEW

File: src/stores/AccountsStore.js [hunk 1]
```diff
@@ -296,6 +296,8 @@ export default class AccountsStore extends Container<AccountsStoreState> {
 
 	async saveNewIdentity(seedPhrase, pin) {
 		const updatedIdentity = deepCopyIdentity(this.state.newIdentity);
+		//TODO encrypt seedPhrase with password in the future version,
+		// current encryption with only seedPhrase is compatible.
 		updatedIdentity.encryptedSeed = await encryptData(seedPhrase, pin);
```
Comment (hanwencheng) [inline src/stores/AccountsStore.js]: Currently we only encrypt `seedPhrase` in `encryptedSeed`, but will include `password` in the future

File: src/stores/types.js [hunk 2]
```diff
@@ -38,7 +38,8 @@ type AccountMeta = {
 };
 
 export type Identity = {
-	encryptedSeedPhrase: string,
+	// encrypted seed include seedPhrase and password
```
Comment (hanwencheng) [inline src/stores/types.js]: currently only seedPhrase

File: .gitignore [hunk 3]
```diff
@@ -45,6 +45,7 @@ build/
 .gradle
 local.properties
 *.iml
+gen/
```
Comment (yjkimjunior) [inline .gitignore]: just curious, what is generating this?

Comment (hanwencheng) [reply .gitignore]: I don't know either, after build android with `gradle assemble`, it created there files: `buildConfig.java`, `Manifest.java` and `R.java`. On each of them there is comments said: 

```
/*___Generated_by_IDEA___*/
```

So I just add them there.

**Pull request merged**